src/items.rs

```diff
@@ -2161,18 +2228,20 @@ impl Rewrite for ast::Param {
                 shape,
                 has_multiple_attr_lines,
             )
+            .ok_or_else(|| RewriteError::Unknown)
```

Comment:
might also want to see how difficult it would be to change the signature of `rewrite_explicit_self`